### PR TITLE
Ui theme update

### PR DESCRIPTION
**Pull request by vpereverzev**
inspector dark theme examples:
Peek 2020-06-17 13-04
Peek 2020-06-17 13-05
Peek 2020-07-03 13-34

inspector light theme example:
Peek 2020-07-03 13-38

mu4 prototype (dark theme):
Peek 2020-07-03 15-08

### CODE REVIEW

**Comment by igorkorsukov** [comment]
> `framework/uicomponents/qml/MuseScore/UiComponents/CheckBox.qml`: I think this is unnecessarily too long. It'd be very inconvenient if you try to type that whole path in git. Why not just `framework/uicomponents/qml/CheckBox.qml`? Is there anything that will not be in "MuseScore"? Why duplicating `UiComponents`?
> 
> Or is this because of the `MuseScore.UiComponents` module used in QML? Even so, must we create so many subfolders subsequently?

Yes, this is due to the specificity of the implementation of Qml modules

**Comment by Jojo-Schmitz** [comment]
You have a mix of commit messages in past tense and present tense, "Removed", Got did "... vs.  "Update". IIRC the standard was to use present tense only?

File: framework/ui/view/qmltheme.cpp [hunk 1]
```diff
@@ -19,26 +19,150 @@
...
+{
+    return currentThemeProperites().value(BUTTON_OPACITY_HOVER).toReal();
+}
+
+qreal QmlTheme::buttonOpacityHit() const
+{
+    return currentThemeProperites().value(BUTTON_OPACITY_HIT).toReal();
+}
+
+QHash<int, QVariant> QmlTheme::currentThemeProperites() const
```
Comment (igorkorsukov) [inline framework/ui/view/qmltheme.cpp:160]: Confuses what is passed by value.

File: mu4/appshell/qml/HomePage/HomeMenu.qml [hunk 2]
```diff
@@ -1,26 +1,74 @@
...
+        spacing: 0
+
+        model: [
+            { "name": "scores", "title": "Scores", "icon": IconCode.MUSIC_NOTES },
+            { "name": "extensions", "title": "Add-ons", "icon":  IconCode.LIST_ADD },
+            { "name": "audio", "title": "Audio & VST", "icon":  IconCode.AUDIO },
+            { "name": "feautured", "title": "Featured", "icon":  IconCode.STAR },
+            { "name": "learn", "title": "Learn", "icon":  IconCode.GRADUATION_CAP },
+            { "name": "support", "title": "Support", "icon": IconCode.FEEDBACK },
+            { "name": "account", "title": "Account", "icon":  IconCode.ACCOUNT }
```
Comment (igorkorsukov) [inline mu4/appshell/qml/HomePage/HomeMenu.qml:29]: Need to add string translatability (qsTrc)

**Comment by igorkorsukov** [comment]
Also need to add the styling of the QWidget dock component (panel headers, tabs)

**Comment by igorkorsukov** [comment]
And it would be great to make a page in devtools, with a display of colours and a description of what it is used for.

**Comment by vpereverzev** [comment]
> Also need to add the styling of the QWidget dock component (panel headers, tabs)

agreed

**Comment by vpereverzev** [comment]
> And it would be great to make a page in devtools, with a display of colours and a description of what it is used for.

Yep, I've also thought about it, I'm going to make it in the separate PR

File: mu4/appshell/qml/HomePage/HomeMenu.qml (hunk 2 above)
Comment (vpereverzev) [reply mu4/appshell/qml/HomePage/HomeMenu.qml]: done

**Pull request merged**